File: golang-hello-world/hello2/hello2.go
package main

import (
	"fmt"

	"github.com/hashgraph/hedera-sdk-go/v2"
)

func main() {
	fmt.Println("hello world (2)")
	accId, err := hedera.AccountIDFromString("0.0.12345")
	if err != nil {
		panic(err)
	}
	fmt.Println(accId)
}

File: java-hello-world/hello2/src/main/java/hello2/Hello2.java
package hello2;

import com.hedera.hashgraph.sdk.AccountId;

public class Hello2 {
    public static void main(String[] args) {
        System.out.println("Hello World! (2)");
        AccountId accId = AccountId.fromString("0.0.12345");
        System.out.println(accId);
    }
}
